### PR TITLE
Revert "add finish markdown story -new branch"

### PR DESCRIPTION
**Pull request by ZoeyHashemi**
Reverts ZoeyHashemi/Choose-your-own-adventure#2

### CODE REVIEW

File: black-eyes-adventure.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
 Black Eyes, feeling brave after the encounter with the wolf, decided to go on a solo adventure. As he ventured into the forest, he heard a rustling in the bushes...
-Black Eyes, feeling brave after the encounter with the wolf, decided to go on a solo adventure. As he ventured into the forest, he heard a rustling in the bushes. His heart pounded, but he knew he had to face whatever lay ahead.
-
-Suddenly, to his surprise, it was not a stranger but his mother, the goat, emerging from the bushes. She looked worried but relieved to see him.
-
-"My child," she said, "I’ve been searching for you. The forest is dangerous, and I fear the wolf may not be the only threat here. We must decide what to do next."
-
-What will Black Eyes do next?
-- **Listen to his mother and return home safely**: [Go to Goat Returns](goat-returns.md)
-- **Convince his mother to continue the adventure**: [Go to Continue Adventure](continue-adventure.md)
```
Comment (ZoeyHashemi) [inline black-eyes-adventure.md:12]: @mahm0106 please review this PR

**Review by mahm0106 — APPROVED** [review]
Looks good

**Comment by ZoeyHashemi** [comment]
we have to repeat this action once again (stage 4 at the assignment)

________________________________________

**Comment by ZoeyHashemi** [comment]
I've added the new option to your story. follow the instruction of the stage 4 & 5

________________________________________

**Pull request merged**